StripeFinancialConnections/StripeFinancialConnections/Source/Native/PartnerAuth/PartnerAuthViewController.swift

```diff
@@ -25,7 +25,18 @@ protocol PartnerAuthViewControllerDelegate: AnyObject {
 
 @available(iOSApplicationExtension, unavailable)
 final class PartnerAuthViewController: UIViewController {
-    
+
+    /**
+     Unfortunately there is a need for this state-full parameter. When we get url callback the app might not be in foreground state.
+     If we then authorize the auth session will fail as you can't do background networking without special permission.
+     */
+    private var unprocessedReturnURL: URL?
+    private var subscribedToURLNotifications = false
+    private var subscribedToAppActiveNotifications = false
+    private var continueStateView: ContinueStateView?
+
+    private var currentAuthSession: FinancialConnectionsAuthSession?
```

Comment:
note that there is a `pendingAuthSession`:
- https://github.com/stripe/stripe-ios/blob/f276c682/StripeFinancialConnections/StripeFinancialConnections/Source/Native/PartnerAuth/PartnerAuthDataSource.swift#L35

not sure if that is useful but maybe they can be combined (not claiming they should, just opening the possibility if it makes sense)

Reply:
Yeah I saw that but it was private to data manager... I guess could make sense to open it up and make it read only